Jenkinsfile.nightly-build

```diff
@@ -33,19 +33,19 @@ pipeline {
     }
     stages {
     	stage("Prepare workspace") {
-    		steps {
-    			library identifier: 'lisk-jenkins@nh-lisk-build', retriever: modernSCM(
-                 	[$class: 'GitSCMSource',
-                 	remote: 'https://github.com/LiskHQ/lisk-jenkins.git'])
-    		}
+            steps {
+            	library identifier: 'lisk-jenkins@master', retriever: modernSCM(
+                    [$class: 'GitSCMSource',
+                    remote: 'https://github.com/LiskHQ/lisk-jenkins.git'])
+            }
     	}
         stage('Build release') {
             steps {
             	script {
-					def release = liskBuild(branch: '1.0.0', network: 'dev')
```

Comment:
@abelboldu Why its build against the `dev` net? It should be `main` net.

Reply:
I don't think it has any effect anyway, but we are building nightlies from the `devnet-development` branch, so I think the network should be `dev`

Reply:
If the branch is parameterized or env variable then the network should also be passed in same approach. Instead of hardcoding it in the file.

Reply:
1/ `env.BRANCH_NAME` is the branch Jenkins is currently working on, there is no need for a parameter.
2/ I would argue nightly builds will always be `dev` builds (and not `test` or `main`)